flaskblog/routes.py

```diff
@@ -141,4 +149,71 @@ def delete_post(post_id):
     db.session.delete(post)
     db.session.commit()
     flash('Your post has been deleted!', 'danger')
-    return redirect(url_for('home'))
\ No newline at end of file
+    return redirect(url_for('home'))
+
+
+
+# add sample data to POSTS
+path = "/Users/erdoganb/Desktop/erdo-dev/flask_blog/flaskblog/static/blog_posts.json"
```

Comment:
Use relative path instead of absolute path. This code doesn't work anywhere than your local.